### ISSUE TITLE
Response types do not play well with disallow_any_expr=true

### ISSUE THREAD

**Issue by lucas-flowers** [vemel/mypy_boto3_builder#103]
**Describe the bug**

In our project, we have dynamic types completely disabled unless you specifically ignore them. This currently works very well, but when trying to upgrade to the latest version, we run into trouble due to an `Any` type deep inside response type defs.

**To Reproduce**

1. Install (for example) 'boto3-stubs[ssm]'

2. Run `mypy` on the following code sample with `disallow_any_expr = true` in `mypy.ini`:

```python
import boto3

result = boto3.client("ssm").get_parameters_by_path(Path="/parameter")
```

3. Actual mypy output:

```
test.py:3: error: Expression type contains "Any" (has type "GetParametersByPathResultTypeDef")  [misc]
Found 1 error in 1 file (checked 1 source file)
```

**Expected mypy-output**

```
Success: no issues found in 1 source file
```

The culprit appears to be some `Any` type buried inside responses that was added in an update since we last pinned our versions.

This also affects responses from SQS (which is the more annoying issue for us, though the SSM example above was a little simpler to write down).

**Additional context**

The example above uses:

```
boto3==1.18.13
boto3-stubs==1.18.13
mypy==0.910
mypy-boto3-ssm==1.18.13
```

Our project does not have this problem (because we haven't upgraded yet due to this). The versions are currently pinned to:

```
boto3==1.17.99
boto3-stubs==1.17.99
mypy==0.812
mypy-boto3-ssm==1.17.99
```

While we can just add some `type: ignore` comments, it would be more preferable if boto3-stubs could give more specific types than `Any` in those responses if at all possible. If there's a workaround I'm not aware of, that would also help!

**Comment by vemel** [comment]
Thank you for the report!

I am going to test it and let you know. I am currently using `mypy 0.910`.

**Comment by vemel** [comment]
As of now, I do not have a proper fix for the issue. Looks to me like a bug in `mypy`, since `pyright` works as it should.

If you have any ideas, please let me know.

**Comment by vemel** [comment]
The latest `mypy-boto3-builder 6.1.0` release has a fix for the issue. So you can expect `boto3-stubs 1.19.10` that should solve your issue. The original issue was improperly typed `ResponseMetadata` that affected all services.

Let me know if it works for you as expected or if you still have any issues. Thank you again for the report and your patience!
